pahfit/base.py

```diff
@@ -564,3 +569,69 @@ def read(self, filename, tformat=None):
         readout = (bb_info, df_info, h2_info, ion_info, att_info)
 
         return readout
+
+    def starting_point(self, obs_x, obs_y, param_info):
+        """
+        return updated starting point in param_info based on the input spectrum
```

Comment:
This is a very minimal docstring.  It would be great if this could be expanded to the more standard format like that at the very start of the class.

Reply:
Docstring has been added.